osu.Game/Overlays/OverlayPanelDisplayStyleControl.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osuTK;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics.UserInterface;
+using osu.Framework.Allocation;
+using osuTK.Graphics;
+using osu.Framework.Graphics.Cursor;
+
+namespace osu.Game.Overlays
+{
+    public class OverlayPanelDisplayStyleControl : OsuTabControl<OverlayPanelDisplayStyle>
+    {
+        protected override Dropdown<OverlayPanelDisplayStyle> CreateDropdown() => null;
+
+        protected override TabItem<OverlayPanelDisplayStyle> CreateTabItem(OverlayPanelDisplayStyle value) => new PanelDisplayTabItem(value);
+
+        protected override bool AddEnumEntriesAutomatically => false;
+
+        public OverlayPanelDisplayStyleControl()
+        {
+            AutoSizeAxes = Axes.Both;
+
+            AddTabItem(new PanelDisplayTabItem(OverlayPanelDisplayStyle.Card)
+            {
+                Icon = FontAwesome.Solid.Square
+            });
+            AddTabItem(new PanelDisplayTabItem(OverlayPanelDisplayStyle.List)
+            {
+                Icon = FontAwesome.Solid.Bars
+            });
+        }
+
+        protected override TabFillFlowContainer CreateTabFlow() => new TabFillFlowContainer
+        {
+            AutoSizeAxes = Axes.Both,
+            Direction = FillDirection.Horizontal
+        };
+
+        private class PanelDisplayTabItem : TabItem<OverlayPanelDisplayStyle>, IHasTooltip
```

Comment:
These should probably have `HoverClickSounds` for UX consistency, to indicate interactability.

Reply:
oops, true